documentation/api/definitions.md

```diff
@@ -87,6 +87,48 @@ Enumeration that describes the current state of the collection rule.
 | `Throttled` | Indicates that the collection rule is temporarily throttled because the ActionCountLimit has been reached within the ActionCountSlidingWindowDuration. |
 | `Finished` | Indicates that the collection rule has completed and will no longer trigger. |
 
+## CapturedMethod
+
+First Available: 9.0 Preview 3
+
+Object describing a captured method and its parameters.
+
+| Name | Type | Description |
+|---|---|---|
+| `requestId` | guid | Unique identifier for the capture request operation. |
+| `activityId` | string | An identifier for the current activity at the time of the capture. For more information see [Activity.Id](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.activity.id).|
+| `capturedDateTime` | DateTime | Time when the method call was captured. |
+| `module` | string | The method module name. |
+| `type` | string | The method type name. |
```

Comment:
We should also be recording `threadId` and `activityIdFormat` to preserve parity with the current set of ambient data that parameters collect.

The current set of ambient data is defined here https://github.com/dotnet/dotnet-monitor/blob/40ff7284fae2b72728d7a9025be021c682420457/src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/ParameterCapturingLogger.cs#L20

And collected here:  
https://github.com/dotnet/dotnet-monitor/blob/40ff7284fae2b72728d7a9025be021c682420457/src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/ParameterCapturingLogger.cs#L104